lib/ably/models/idiomatic_ruby_wrapper.rb

```diff
@@ -35,9 +36,6 @@ module Ably::Models
   #   ruby_hash[:none] # => nil
   #   ruby_hash.none # => nil
   #
-  # @!attribute [r] stop_at
-  #   @return [Array<Symbol,String>] array of keys that this wrapper should stop wrapping at to preserve the underlying Hash as is
```

Comment:
Is there a reason we're removing these? I think if we're not replacing them they should probably be kept

Reply:
IMO, `@!attribute [r] stop_at` and related are not so important. Because it's obvious for someone who read the code that there is stop_at attribute, so I focused on removed such yard comments. Also, it's not generated in the docs.

Okay, reverted.